<launch>
    <group ns="tj2" >
        <node name="tj2_yolo" pkg="tj2_yolo" type="tj2_yolo_node.py" output="screen" required="false">
            <param name="model_path" value="$(find tj2_yolo)/models/cargo_2022.pt"/>
            <param name="publish_overlay" value="false"/>
            <param name="report_loop_times" value="false"/>
            <param name="use_depth" value="true"/>
            <param name="marker_persistance" value="0.25"/>
            <remap from="detections" to="cargo/detections"/>
            <remap from="color/image_raw" to="/camera/color/image_raw"/>
            <remap from="depth/image_raw" to="/camera/aligned_depth_to_color/image_raw"/>
            <remap from="color/camera_info" to="/camera/color/camera_info"/>
            <rosparam param="z_depth_estimations" file="$(find tj2_yolo)/config/depth_estimations.yaml" command="load"/>
        </node>
    </group>
</launch>
